Assets/BossRoom/Scripts/Shared/ApplicationController.cs

```diff
@@ -112,15 +112,18 @@ private bool OnWantToQuit()
             return canQuit;
         }
 
-        public void LeaveSession()
+        public void LeaveSession(bool UserRequested)
```

Comment:
mmm I thought I saw this somewhere, looks like I didn't track my things properly and didn't cherrypick part of that fix here https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/562/files?file-filters%5B%5D=.cs&show-viewed-files=true

Reply:
Ah yes I remember that! I can include this part here if it makes sense

Reply:
If you have some time, yeah that'd be great